src/applications/static-pages/facilities/actions/index.js

```diff
@@ -37,7 +37,7 @@ export function fetchFacility(id) {
     dispatch(fetchFacilityStarted());
 
     // eslint-disable-next-line consistent-return
-    return apiRequest(`/facilities/va/${id}`)
+    return apiRequest(`/facilities/va/${id}`, {}, null, null, 'v1')
```

Comment:
Would be nice if there were just a single second argument for `apiRequest` where you could provide all options instead of having to write out all these other unused arguments 🙂